cpp/benchmarks/CMakeLists.txt

```diff
@@ -56,6 +55,14 @@ function(ConfigureBench CMAKE_BENCH_NAME)
         PRIVATE cudf_benchmark_common cudf_datagen benchmark::benchmark_main)
 endfunction()
 
+function(ConfigureNVBench CMAKE_BENCH_NAME)
```

Comment:
Where is this used?

Reply:
The idea behind this PR is to finally replace all current benchmarks (which use google benchmark) by using `nvbench`. I'm working on the first use case of this macro so it's not used yet.